### DISCUSSION TITLE
Can I use Third-Party Exercises as  teaching material ?

### DISCUSSION THREAD

**Discussion by sotanengel** [rust-lang/rustlings#2149] [Q&A] [answered]
@mo8it 
Thank you for your support as always !

I would like to use [Rustlings-jp](https://github.com/sotanengel/rustlings-jp) as a teaching material for [Udemy](https://www.udemy.com/).
Is there any problem??

**Comment by mo8it** [comment, marked as answer]
Go ahead! I have used Rustlings for courses at my university and I highly recommend integrating it when teaching.
Just give credits and don't sell Rustlings itself as a product.

**Comment by sotanengel** [comment]
@mo8it 
Thank you for your response!
I use it only as a teaching material🙇
